src/app/client/src/app/modules/workspace/components/content-editors/collection-editor/collection-editor.component.ts

```diff
@@ -40,7 +40,8 @@ export class CollectionEditorComponent implements OnInit, OnDestroy {
   public ownershipType: Array<string>;
   public queryParams: object;
   resource_framework: string;
-  collectionEditorCDN: string = (<HTMLInputElement>document.getElementById('collectionEditorCDN')).value;
+  collectionEditorCDN: string = (<HTMLInputElement>document.getElementById('collectionEditorCDN')).value ?
```

Comment:
no please change this to 
(<HTMLInputElement>document.getElementById('collectionEditorCDN'))? (<HTMLInputElement>document.getElementById('collectionEditorCDN')).value : ''